game/scripts/npc/abilities/ember_spirit_searing_chains.txt

```diff
@@ -40,12 +40,12 @@
 			"02"
 			{
 				"var_type"						"FIELD_INTEGER"
-				"radius"						"400"
+				"radius"						  "400"
 			}
 			"03"
 			{
 				"var_type"						"FIELD_INTEGER"
-				"total_damage"					"90 160 230 300 450 1000" //OAA
+				"total_damage"					"90 160 230 300 650 1000" //OAA
```

Comment:
can you make this number divisible by 3? 660 is fine. It'd be 100 dmg per second at 4 than 220 at 5

Reply:
Has new scaling "660 1020" Now both divisible by 3 and 4 for with/without talent. :)